Tests/Benchmarks/UsersOfGetClosedGenericInterfaces.cs

```diff
@@ -166,7 +165,7 @@ public void GlobalSetup()
             }
 
             context = new Context();
-            config = new Config();
+            new Config();
```

Comment:
Have a look at the changes in 684b5782b79d0f50094e61451dc77eaa5bc27647

`IEquivalencyAssertionOptions` is now a member of `IEquivalencyValidationContext`.
So I would just make `Context.Options` writable and initialize it as
```cs
context = new Context()
{
    Options = new Config()
};
```